client/app/components/viz/VariantInspectCard.vue

```diff
@@ -2048,10 +2055,10 @@ export default {
     mosaicValuesMap() {
       const mosaicValues = [];
       for (const item of this.selectedMosaicVariantAnnotations) {
-        if (this.mosaicVariant && this.mosaicVariant.hasOwnProperty(item.value)) {
+        if (this.mosaicVariantObject && this.mosaicVariantObject.hasOwnProperty(item.value)) {
           let value = "";
-          if (this.mosaicVariant[item.value].length > 0){
-            value = this.mosaicVariant[item.value].join(", ");
+          if (this.mosaicVariantObject[item.value].length > 0){
```

Comment:
This code probably won't work in the scenario where length is > 0, but it is not an array. For example, a string value has a length > 0. You can check for the type of array using isArray, which ensures that the join (an array method) will work.